level_5/interpreter.hpp

```diff
@@ -0,0 +1,81 @@
+#pragma once
+#include <iostream>
+#include <stdexcept>
+#include "lexer.hpp"
+
+class Interpreter{
+	private:
+		Lexer lexer_;
+		Token current_token_;
+		void error_(){
```

Comment:
Shortcut: Spacing all over the place is a bit weird. Have a look at the other sources, and try to mimmick this. :)